lib/cdo/shared_constants.rb

```diff
@@ -210,7 +210,7 @@ module SharedConstants
       "led.pulse": null,
       "buzzer.frequency": null,
       "buzzer.note": null,
-      "buzzer.off": null,
```

Comment:
This list populates the App Lab sandbox mode toolbox, where we only want to include `buzzer.stop()` per the toolbox unification decision.